desktop-widgets/chartwidget.h

```diff
@@ -0,0 +1,26 @@
+#ifndef CHARTWIDGET_H
+#define CHARTWIDGET_H
+
+#include <QWidget>
+#include "ui_chartwidget.h"
+#include "qt-models/datestatsmodel.h"
+
+namespace Ui {
+class ChartWidget;
```

Comment:
This forwad declaration makes no sense anymore, now that you include `ui_chartwidget.h`.